backend/tests/test_validators.py

```diff
@@ -28,3 +34,58 @@ def test_enum_validator_skips_validation_when_context_set():
         TestModel.model_validate(
             {"value": "invalid"}, context={"skip_validation": True}
         )
+
+
+@pytest.mark.parametrize(
+    "filename,expected",
+    [
+        ("wikipedia_en_2024-01.zim", does_not_raise()),
```

Comment:
`wikipedia_en_2024-01.zim`, `ted-talks_eng_2024-03.zim` and `ted-talks_eng_{period}.zim` are all invalid (they miss the "selection" part).

`wikipedia_en_all_2024-01.zim`, `ted-talks_eng_football_2024-03.zim` and `ted-talks_eng_football_{period}.zim` are all valid (selection)

`wikipedia_en_all_nopic_2024-01.zim` is valid as well (selection + flavor)

Nota: you should also fix other invalid tests below to have only one validation issue, e.g. replace `wikipedia_eng_2024-01` (which misses the selection and the .zim extension) with `wikipedia_eng_all_2024-01` (which misses only the .zim extension).